lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -876,12 +876,12 @@ def _cve(
     dt : float
         Time step
     blur_constant : float
-        Motion blur coefficient. Should be between 0 and 1/4.
+        Motion blur coefficient. Should be between 0 and 1/4 or `np.nan` if not clearly defined.
```

Comment:
in the docstring for `Kymo.motion_blur` you say it should be between 0 and 1/6 but here 0 and 1/4 ??

Reply:
It's not inconsistent.

1/6 `t_exposure` / `t_frame` is what you get for a continuously open shutter, but you can do worse than that.

If you look at the integral, you'll find that the worst you can get is a two pulse shutter per frame. So basically taking to snapshots at the start and end of the frame, which results in 1/4.

Reply:
ok yeah, it is a bit confusing here but I agree could be just as confusing/distracting to have all this extra information that isn't particularly relevant to the data we expect. We can always update if we start getting questions about this